src/stores/RoomListStore.js

```diff
@@ -54,6 +54,7 @@ class RoomListStore extends Store {
                 "im.vector.fake.archived": [],
             },
             ready: false,
+            roomCache: {}, // roomId => { cacheType => value }
```

Comment:
a little bit more commentary about how the cache is used would be good (e.g. what the values are for the different types of caching; what the lifetime is of the entries; etc)